sunpy/net/dataretriever/sources/kanzelhohe.py

```diff
@@ -0,0 +1,128 @@
+import datetime
+import numpy as np
+
+import astropy.units as u
+
+from sunpy.net.dataretriever.client import GenericClient
+from sunpy.util.scraper import Scraper
+from sunpy.net import attrs as a
+
+__all__ = ['KanzelhoheClient']
+
+
+class KanzelhoheClient(GenericClient):
+    """
+    Returns a list of URLS to Kanzelhohe H-alpha files corresponding to value of input timerange.
```

Comment:
```suggestion
    Returns a list of URLS to Kanzelhöhe H-alpha files corresponding to value of input timerange.
```
We should be able to put ö on docstrings.